GDJS/GDJS/Extensions/Builtin/AdvancedExtension.cpp

```diff
@@ -210,12 +220,13 @@ AdvancedExtension::AdvancedExtension() {
             codeGenerator.GenerateUpperScopeBooleanFullName("isConditionTrue", context);
 
         return resultingBoolean + " = " +
-               gd::String(instruction.IsInverted() ? "!" : "") +
+               gd::String(instruction.IsInverted() ? "!" : "") + "(" +
```

Comment:
The opening parenthesis was in the `leftOperand` parameter. While it was working, it's now easier to follow.